### PR TITLE
#7792 heading style

### LINKED ISSUE #7792: Mint heading style
image
- the heading should look like this
image

- in dark mode it could have the same fill as the background (no gradient) - while having stroke and the typical dropshadow 

figma: https://www.figma.com/file/JXNtryaNYhY0XJNArzvPQx/Handoff?type=design&node-id=142%3A1572&mode=design&t=GKkayTA5Y8dTj3H1-1

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Needs QA check

- @kodadot/qa-guild please review

## Context

- [x] Closes #7792
- [ ] Requires deployment <snek/rubick/worker>

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=13Qx65nLd6SwdtjrRyuoEtp9CKXhF651xdHBPaXcvhwKm4N1)

## Screenshot 📸

- [x] My fix has changed UI
Screen Shot 2023-10-23 at 10 36 53 PM

### CODE REVIEW

**Comment by prachi00** [comment]
https://stackoverflow.com/questions/69253420/text-stroke-webkit-text-stroke-css-problem

couldn't find a better solution, -webkit-text-stroke was causing this weird shapes problem and there is no other way around it

**Review by roiLeo — COMMENTED** [review]
I think we already had this issue in the past, check how it's done on landing page "Polkadot NFT" text

**Comment by prachi00** [comment]
> I think we already had this issue in the past, check how it's done on landing page "Polkadot NFT" text

I did, it used text-shadow and white color, with white color, you can't apply gradient, removing white color makes the text all black

**Review by preschian — APPROVED** [review]

**Comment by prachi00** [comment]
> no gradient here for me: white mode: image no gradient/drop shadow dark mode: image

isnt gradient only suppose to be there when toggle is on?

**Comment by prury** [comment]
> > no gradient here for me: white mode: image no gradient/drop shadow dark mode: image
> 
> isnt gradient only suppose to be there when toggle is on? @prury

image

**Comment by prachi00** [comment]
> > > no gradient here for me: white mode: image no gradient/drop shadow dark mode: image
> > 
> > 
> > isnt gradient only suppose to be there when toggle is on? @prury
> 
> @prachi00 yes, but as you can see below, gradient is not present when toggle is on: image

oh check now, my bad

File: assets/styles/pages/create.scss [hunk 1]
```diff
@@ -48,16 +48,34 @@
...
+  pointer-events: none;
+  color: transparent;
   background: linear-gradient(to right, white, white, white);
+  -webkit-background-clip: text;
   background-clip: text;
-  -webkit-text-fill-color: transparent;
-  -webkit-text-stroke: 1px black;
-  filter: drop-shadow(2px 2px black);
-  font-weight: bold;
+  font-weight: 600;
```
Comment (roiLeo) [inline assets/styles/pages/create.scss:72]: `has-text-weight-semibold`

**Review by roiLeo — APPROVED** [review]
otherwise lgtm

**Comment by yangwao** [comment]
thanks!
pay 20 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $20 @ 4.34 USD/DOT ~ 4.608 $DOT
🧗 [13Qx65nLd6SwdtjrRyuoEtp9CKXhF651xdHBPaXcvhwKm4N1](https://polkadot.subscan.io/account/13Qx65nLd6SwdtjrRyuoEtp9CKXhF651xdHBPaXcvhwKm4N1)
🔗 [0x8e035e9bd95e0f243716d61a6aff50cdaea6a21ae1db7973af6e154d654295f9](https://polkadot.subscan.io/extrinsic/0x8e035e9bd95e0f243716d61a6aff50cdaea6a21ae1db7973af6e154d654295f9)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**